src/main/scala/centaur/test/standard/CentaurTestFormat.scala

```diff
@@ -13,6 +13,7 @@ sealed abstract class CentaurTestFormat(val name: String) {
     case WorkflowFailureTest => "fail during execution"
     case RunTwiceExpectingCallCachingTest => "call cache the second run of"
     case RunTwiceExpectingNoCallCachingTest => "NOT call cache the second run of"
+    case RunFailingTwiceExpectingNoCallCachingTest => "NOT call cache the second run of"
```

Comment:
"NOT call cache the second failing run of" maybe to differentiate it from the other test spec of RunTwiceExpectingNoCallCachingTest